guides/small_prefixes.md

```diff
@@ -50,3 +50,24 @@ prefix set too_small; {
     return true;
 }
 ```
+
+## Cisco classic IOS and IOS XE
+```
+ip prefix-list peerfilter seq 5 deny 0.0.0.0/0
+ip prefix-list peerfilter seq 10 permit 0.0.0.0/0 ge 8 le 28
```

Comment:
No my mistake, the examples don't - but still I think le 27 makes more sense. But I can change it anyhow.